open_samus_returns_rando/specific_patches/map_icons.py

```diff
@@ -29,24 +29,21 @@ def patch_tiles(editor: PatcherEditor):
             for tile_idx in range(len(tiles)):
                 current_tile = tiles[tile_idx]
 
-                # Items
                 icons = current_tile["icons"]
-                if len(icons) != 0 and (
-                    icons[0]["icon"].endswith("tank")
-                    or icons[0]["icon"].startswith("itemsphere")
-                ):
-                    if current_tile["tile_type"] == TileType.HEAT:
-                        icons[0]["icon"] = "itemenabledheat"
-                    else:
-                        icons[0]["icon"] = "itemenabled"
-
-                for door in doors:
-                    if len(icons) == 1 and icons[0]["actor_name"].startswith(door):
-                        icons[0]["clear_condition"] = ""
-                        if "left" in icons[0]["icon"]:
-                            icons[0]["icon"] = "doorpowerleft"
+                if len(icons) != 0:
+                    icons_length = icons[0] if len(icons) == 1 else icons[1]
```

Comment:
Can we rename that variable? It is very confusing for me to name something `...length` and it is not a number but a dict instead. (It's cleary not a `length` at all?!)

Reply:
Sure